### PR TITLE
docs(core): update angular to nx guide

### PR DESCRIPTION
**Pull request by isaacplmann**
Updates the angular cli to Nx guide

It accurately describes how to use `ng add @nrwl/angular`.  But should we direct people to use `nx init` instead?  I think people coming from Angular will be more comfortable with the `ng add` syntax, so I left the guide using that syntax.

### CODE REVIEW

File: docs/shared/migration/migration-angular.md [hunk 1]
```diff
@@ -205,58 +204,50 @@ yarn create nx-workspace myorg --preset=angular
...
+├── src/
+│   ├── app/
+│   ├── assets/
+│   ├── favicon.ico
+│   ├── index.html
+│   ├── main.ts
+│   ├── styles.css
+│   └── test-setup.ts
 ├── .eslintrc.json
+├── .eslintrc.<app name>.json
```
Comment (FrozenPandaz) [inline docs/shared/migration/migration-angular.md:226]: This should be `.eslintrc.base.json`

File: docs/shared/migration/migration-angular.md [hunk 2]
```diff
@@ -367,8 +358,8 @@ If you are using `Protractor` for E2E testing:
     "lint": {
       "executor": "@angular-devkit/build-angular:tslint",
```
Comment (FrozenPandaz) [inline docs/shared/migration/migration-angular.md:359]: Is this still valid?

File: docs/shared/migration/migration-angular.md [hunk 3]
```diff
@@ -205,58 +204,50 @@ yarn create nx-workspace myorg --preset=angular
 
 When prompted for the `application name`, enter the _project name_ from your current `angular.json` file.
```
Comment (FrozenPandaz) [inline docs/shared/migration/migration-angular.md:205]: Above this, it says to use `--preset angular`, it should now be `--preset angular-standalone`.

Or honestly.. maybe `--preset empty`?

File: docs/shared/migration/migration-angular.md [hunk 4]
```diff
@@ -188,75 +187,67 @@ To start, run the command to generate an Nx workspace with an Angular applicatio
...
 ```
 
 ### Copying over application files
 
 Your application code is self-contained within the `src` folder of your Angular CLI workspace.
 
 - Copy the `src` folder from your Angular CLI project to the `apps/<app name>` folder, overwriting the existing `src` folder.
-- Copy any project-specific configuration files, such as `browserslist`, or service worker configuration files into their relative path under the `apps/<app name>` folder.
-- Transfer the `assets`, `scripts`, `styles`, and build-specific configuration, such as service worker configuration, from your Angular CLI `angular.json` to the `apps/<app name>/project.json` file.
+- Copy any project-specific configuration files, such as `browserslist`, or service worker configuration files into their relative path in the root of the repo.
```
Comment (FrozenPandaz) [inline docs/shared/migration/migration-angular.md:249]: This should say to move it to the `src` in the root or something.

File: docs/shared/migration/migration-angular.md [hunk 5]
```diff
@@ -276,29 +267,29 @@ ng test <app name>
 
 If you are using `Karma` for unit testing:
 
-- Copy the `karma.conf.js` file to your `apps/<app name>` folder.
-- Copy the `test.ts` file to your `apps/<app name>/src` folder.
-- Copy the `test` target in your `architect` configuration from your Angular CLI `angular.json` file into the `targets` configuration in the `apps/<app name>/project.json` file in your Nx workspace.
+- Copy the `karma.conf.js` file to the root folder.
+- Copy the `test.ts` file to your `src` folder.
+- Copy the `test` target in your `architect` configuration from your Angular CLI `angular.json` file into the `targets` configuration in the `project.json` file in your Nx workspace.
 - Update the `test` target file paths to be relative to `apps/<app name>`.
```
Comment (FrozenPandaz) [inline docs/shared/migration/migration-angular.md:273]: This line isn't needed anymore.

File: docs/shared/migration/migration-angular.md [hunk 6]
```diff
@@ -276,29 +267,29 @@ ng test <app name>
...
 
-```jsonc {% fileName="apps/<app name>/project.json" %}
+```jsonc {% fileName="project.json" %}
```
Comment (FrozenPandaz) [inline docs/shared/migration/migration-angular.md:275]: It's a lot more 1:1 now.. I'd say it's not worth showing this example.

The one thing I would say to do is to change `builder => executor`...

File: docs/shared/migration/migration-angular.md [hunk 7]
```diff
@@ -339,20 +330,20 @@ ng e2e <app name>-e2e
 
 If you are using `Protractor` for E2E testing:
 
-- Delete the `apps/<app name>-e2e` folder that was generated to use Cypress.
-- Copy the `e2e` folder from your Angular CLI workspace into the `apps` folder.
+- Delete the `e2e` folder that was generated to use Cypress.
+- Copy the `e2e` folder from your Angular CLI workspace into the root folder.
 - Rename the `e2e` folder to `<app name>-e2e`.
```
Comment (FrozenPandaz) [inline docs/shared/migration/migration-angular.md:335]: This line can be removed.

File: docs/shared/migration/migration-angular.md [hunk 8]
```diff
@@ -339,20 +330,20 @@ ng e2e <app name>-e2e
...
-- Create the project configuration file at `apps/<app name>-e2e/project.json`.
-- Copy the project configuration for `app name` from the Angular CLI workspace `angular.json` file to `apps/<app name>-e2e/project.json` and adjust the file paths to be relative to `apps/<app name>-e2e`.
+- Create the project configuration file at `e2e/project.json`.
+- Copy the project configuration for `app name` from the Angular CLI workspace `angular.json` file to `e2e/project.json` and adjust the file paths to be relative to `e2e`.
 
-```json {% fileName="apps/<app name>-e2e/project.json" %}
+```json {% fileName="e2e/project.json" %}
```
Comment (FrozenPandaz) [inline docs/shared/migration/migration-angular.md:339]: Again this should be more 1:1 now.

I would just mention to change `architect => targets` and `builder => executor`

File: docs/shared/migration/migration-angular.md [hunk 9]
```diff
@@ -365,21 +356,20 @@ If you are using `Protractor` for E2E testing:
...
-    }
+    },
   },
   "implicitDependencies": ["<app name>"],
   "tags": []
 }
 ```
 
-Create a `tsconfig.e2e.json` file under `apps/<app name>-e2e` folder:
+Create a `tsconfig.e2e.json` file under `e2e` folder:
```
Comment (FrozenPandaz) [inline docs/shared/migration/migration-angular.md:370]: We could call this `tsconfig.json`?

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

**Review by leosvelperez — COMMENTED** [review]
The section about `--preserve-angular-cli-layout` (https://github.com/nrwl/nx/blob/master/docs/shared/migration/migration-angular.md?plain=1#L14-L36) probably needs to be amended as well.

Starting in Nx v15.3.0, the changes made to an Angular v15 workspace are different from what is currently listed in that section:

- It'll generate a `project.json`
- The `angular.json` is removed
- `prettier` is installed

Note that those changes only happen for Angular CLI v15 workspaces using `@nrwl/angular@15.3.0`. For Angular CLI v14 workspaces migrating with an older version of the `@nrwl/angular` plugin, the changes will be the ones that are listed today. We can at least call out that the updated list of changes are only for the latest version and prior versions will see some differences.

File: docs/shared/migration/migration-angular.md [hunk 10]
```diff
@@ -2,42 +2,47 @@
...
-ng add @nrwl/angular --preserve-angular-cli-layout
+nx init
 ```
 
-**Note**: If you specify a version of Nx (e.g. `ng add @nrwl/angular@13.10.0`), please make sure to use the appropriate command as shown in the compatibility table below:
+**Note**: To migrate to legacy versions of Nx, you'll need to refer to the following compatibility table below:
 
 | Nx version          | Collection to use | Flag to use                     | Example                                                       |
 | ------------------- | ----------------- | ------------------------------- | ------------------------------------------------------------- |
 | >= 13.10.0          | `@nrwl/angular`   | `--preserve-angular-cli-layout` | `ng add @nrwl/angular@13.10.0 --preserve-angular-cli-layout`  |
```
Comment (leosvelperez) [inline docs/shared/migration/migration-angular.md:17]: Probably we want to add the max in the range for the Nx version here to clearly state up to which version they need to use this. It should be `>= 13.10.0 < 15.3.0`.

File: docs/shared/migration/migration-angular.md [hunk 11]
```diff
@@ -2,42 +2,47 @@
...
+This will enable you to use the Nx CLI in your existing Angular CLI workspace while keeping your existing file structure in place. The following changes will be made in your repo to enable Nx:
 
-- Installs the `nx` and `@nrwl/workspace` packages.
-- Creates an `nx.json` file in the root of your workspace.
-- Adds a `decorate-angular-cli.js` to the root of your workspace, and a `postinstall` script in your `package.json` to run the script when your dependencies are updated. The script forwards the `ng` commands to the Nx CLI (`nx`) to enable features such as [Computation Caching](/concepts/how-caching-works).
+- The `nx` and `@nrwl/workspace` packages will be installed.
+- An `nx.json` file will be created in the root of your workspace.
+- A `decorate-angular-cli.js` file will be added to the root of your workspace, and a `postinstall` script will be added to your `package.json` to run this script when your dependencies are updated. (The script forwards the `ng` commands to the Nx CLI (`nx`) to enable features such as [Computation Caching](/concepts/how-caching-works).)
```
Comment (leosvelperez) [inline docs/shared/migration/migration-angular.md:25]: We also install `prettier` and split the `angular.json` file into `project.json` files.

Comment (leosvelperez) [inline docs/shared/migration/migration-angular.md:25]: Should we also add a note clarifying these changes will be slightly different for legacy versions?

**Review by leosvelperez — APPROVED** [review]
LGTM!

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**